packages/plugins/plugin-deck/src/translations.ts

```diff
@@ -8,6 +8,7 @@ export default [
   {
     'en-US': {
       [DECK_PLUGIN]: {
+        'app name': 'Composer',
```

Comment:
app name is defined by the entrypoint https://github.com/dxos/dxos/blob/main/packages/apps/composer-app/src/plugins.tsx#L188 and is accessible via the theme plugin's translations https://github.com/dxos/dxos/blob/main/packages/plugins/plugin-theme/src/translations/en-US/appkit.ts#L8